src/services/company.js

```diff
@@ -53,51 +53,55 @@ class CompanyService {
     /**
      * @param {*} company_id Id of the company
      * @param {*} showHidden weather to show the company if it is hidden, defaults to false
-     * @param {*} showAdminReason weahter to show the admin reason given to hide this company, defaults to false
+     * @param {*} showAdminReason weather to show the admin reason given to hide this company, defaults to false
      */
-    findById(company_id, showHidden = false, showAdminReason = false) {
+    async findById(company_id, showHidden = false, showAdminReason = false) {
         const query = Company.findById(company_id);
         if (!showHidden) query.withoutBlocked().withoutDisabled();
         if (!showAdminReason) query.hideAdminReason();
-        return query;
+        const company = await query;
+        return  company;
```

Comment:
Is this necessary? I think calling `await findById` has the same effect as it was previously